cmake/impeller_renderer.cmake

```diff
@@ -44,7 +44,10 @@ message(SEND_ERROR
     return()
 endif()
 
-find_package(OpenGL REQUIRED)
+# Some platforms should use GLES
+if(NOT ANDROID AND NOT QNX)
+    find_package(OpenGL REQUIRED)
+endif()
```

Comment:
Yeah this was definitely setup in a way that only works for desktop.

Assuming the toolchain is set up right, I think the "right" way to do this would be to replace `${OPENGL_LIBRARIES}` with `OpenGL::GLES3` for Android and `OpenGL::OpenGL` for desktop targets. But I think this is fine for now.